src/glyphs.js

```diff
@@ -140,4 +176,53 @@ router.post('/:_id/gauges/:index/cache', (req, res) => {
   }
 });
 
+//Used to updated a message attached to a gauge
+router.post('/:_id/gauges/:index/messages/:num', (req, res) => {
+  const processed = processMessageRequest(req);
+
+  if (processed.errors.length > 0) {
+    res.json({
+      'errors': processed.errors
+    });
+  } else {
+    db.collection.updateOne(
+      {
+        _id: req.params._id,
+        [`view.gauges.${req.params.index - 1}.messages.${req.params.num - 1}`]: { $exists: true }
+      },
+      {
+        $set: {
+          [`view.gauges.${req.params.index - 1}.messages.${req.params.num - 1}.text`]: processed.parsed,
```

Comment:
Any desire to set the message to an object and not have to split fields?

Reply:
This question is actually still relevant, as the `processMessageRequest` function handles making sure we don't add extra fields from the request, so we can trust the processed object.